x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/components/precision_slider/precision_slider.tsx

```diff
@@ -54,16 +55,15 @@ export const PrecisionSlider: React.FC = () => {
             {
               defaultMessage: 'Fine tune the precision vs. recall settings on your engine.',
             }
-          )}
-          {/* {' '}
-          <EuiLink>
+          )}{' '}
+          <EuiLink href={`${DOCS_PREFIX}/api-reference.html`} target="_blank">
```

Comment:
Unless I'm missing something from Chris's previous comment, this should be
```suggestion
          <EuiLink href={`${DOCS_PREFIX}/precision-tuning.html`} target="_blank">
```
right? or no?

Reply:
no yeah yeah >_<